packages/gasket-plugin-https/index.js

```diff
@@ -74,12 +75,15 @@ async function start(gasket) {
     onShutdown: one(async function onShutdown() {
       await gasket.exec('onShutdown');
     }),
-    healthChecks: {
-      [healthcheck]: async function healthCheckRequested() {
-        await gasket.exec('healthcheck', HealthCheckError);
+    healthChecks: (function () {
```

Comment:
@kinetifex @runruh-godaddy would this be more aligned with what you were looking for?

Reply:
Yea, avoid the IIFE though for readability, just use a reduce:
```js
async function healthCheckRequested() {
  await gasket.exec('healthcheck', HealthCheckError);
}

healthChecks: routes.reduce((acc, cur) => {
  acc[cur] = healthCheckRequested;
  return acc;
}, {})
```